addons/gen.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+
+set -eux
+
+TEMPLATES="${WD}/../templates"
+DASHBOARDS="${WD}/dashboards"
+TMP=$(mktemp -d)
+
+# create Prometheus template
+helm template prometheus prometheus \
+  --repo https://prometheus-community.github.io/helm-charts \
+  --namespace "replace-me-namespace" \
+  --version 13.2.1 \
+  -f "${WD}/values/prometheus.yaml" \
+  > "${TEMPLATES}/prometheus.yaml"
+
+sed -i'.orig' 's/replace-me-namespace/{{ .Release.Namespace }}/g' "${TEMPLATES}/prometheus.yaml"
```

Comment:
Can we also add a `this file is auto-generated, see addons/gen.sh` to the top of each generated file